automated install/basic-install.sh

```diff
@@ -548,7 +548,7 @@ checkForDependencies() {
     echo ":::"
     echo "::: Checking dependencies:"
 
-  dependencies=( dnsutils bc dnsmasq lighttpd php5-common php5-cgi php5 git curl unzip wget )
+  dependencies=( dnsutils bc dnsmasq lighttpd php-common php-cgi php git curl unzip wget )
```

Comment:
Yeah just looked at the package repos for Ubuntu and aparrently it's not a shared namespace across all releases, need to do OS detection with `lsb_release -a` or such.